index.html

```diff
@@ -155,43 +156,68 @@ <h3 class="Multi-Post_05" >Multi-Post Stories </h3>
-                    </ul>
-                </div>
-        </section>
+        
+      
 </section>
+<section class="AboutcardBack">
+    <div >
+        <h2  class="tonic4" >About Myself </h2>
+        <p class="section1"> Hello I’m a software developer! I can help you build a product , 
+        feature or website Look through some of my work and experience! If you like what you see 
+        and have a project you need coded, don’t hestiate to contact me. </p>
+    </div>
+    <div class = "social-section">
+        <div class="connectmywork">
+            <h4 > LET’S CONNECT</h4>
+        </div>
+        <div class="social">
+            <ul class="social-icons-mywork">
+                <li><a href="#"><img src="./image/Vectortweet.png" alt=""></a></li>
+                <li><a href="#"><img src="./image/VectorM.jpg" alt=""></a></li>
+                <li><a href="#"><img src="./image/Vector5.png" alt=""></a></li>
+                <li><a href="#"><img src="./image/VectorGit.png" alt=""></a></li>
+                <li><a href="#"><img src="./image/Linkedin.svg" alt=""></a></li>
+            </ul>
+        </div>
+    </div>
+    <div class="getme">
+        <button class="button1"><a>Get my resume </a>  </button>
+    </div>
+    <div class="aboutprogram">
+        <ul class="Program-skil">
+            <li  class="Programs-lang" href="#"><img  src="./image/Frame 3.png" alt=""></li>
+            <ul class="Program-skil">
+                <li  class="Programs" href="#"><img src="./image/Frame 9.png" alt=""></li>
+                <li  class="Programs" href="#"><img src="./image/Frame 10.png" alt=""></li>
+                <li  class="Programs" href="#"><img src="./image/Frame 11.png" alt=""></li>
+            </ul>
+            <li  class="Programs-lang" href="#"><img src="./image/Frame 01.png" alt=""></li>
+            <li  class="Programs-lang" href="#"><img src="./image/Frame 02.png" alt=""></li>
+        </ul>
+    </div>
+</section>
+<section class="cardcontact">
+    <form class="cardcontact1" action="https://formspree.io/f/mwkzayzq" method="post" >
+        
+        <div class="sectioncontact">
+            <h2  class="contactme" >Contact me </h2>
+        <p class="contact"> If you have an application you are interested 
+            in developing, a feature that you need built or a project that 
+            needs coding. I’d love to help with it  </p>
+        </div>
+            <div class="form-group">
+                 <input class="texform" type="text"   id="Name"    classe="form-control" placeholder="Enter Your Name" minlength="4" maxlength="30" required />
+                <input class="texform"  type="email"  id="email"  name="email"   placeholder="Enter Your Email"  required  />
+            </div >
+            <div class="form-group2">
+                <textarea class="comment"  name="comment-section" id="comment"  maxlength="500"s  placeholder="Write your message here..."  required ></textarea>
```

Comment:
[OPTIONAL] Kindly check the margin and padding, font family on the Figma template and fix them. e.g. padding-top on the text area. You have 45px and a Figma template, 15px. Please stick to the design for all sizes.
 Figma template:
<img width="631" alt="Screenshot 2022-12-15 at 12 05 18 AM" src="https://user-images.githubusercontent.com/25887253/207714282-00556601-55b4-4006-95a9-d378dafb5e6d.png">

Your design:
<img width="925" alt="Screenshot 2022-12-15 at 12 06 06 AM" src="https://user-images.githubusercontent.com/25887253/207714307-0f033b8e-8c2f-42d3-9168-5a13a21c1382.png">